fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -42,13 +42,13 @@ public class StarOSAgent {
     public static final String SERVICE_NAME = "starrocks";
 
     private StarClient client;
-    private long serviceId;
+    private String serviceId;
     private Map<String, Long> workerToId;
     private Map<Long, Long> workerToBackend;
     private ReentrantReadWriteLock rwLock;
 
     public StarOSAgent() {
-        serviceId = -1;
+        serviceId = "-1";
```

Comment:
if serviceId is changed to string, should the default value be empty() instead of "-1"?

Reply:
Yes, I think so. I have changed it to "".